buildenv/jenkins/variables/defaults.yml

```diff
@@ -90,6 +90,19 @@ build_discarder:
 restart_timeout:
   time: '5'
   units: 'HOURS'
+tests_targets:
```

Comment:
I'm not sure how this works. Lets sit together tomorrow.

Reply:
Those are maps with null values 
see https://learnxinyminutes.com/docs/yaml/